### PR TITLE
feat: load existing asset if present

### PR DESCRIPTION
**Pull request by dcshzj**
## Problem

If the image component has an existing URL field, we are not reflecting that inside the Attachment component.

## Solution

**Breaking Changes**

- [ ] Yes - this PR contains breaking changes
- [x] No - this PR is backwards compatible

**Features**:

- Load the provided URL in the image component if present.

**Improvements**

- Removed the `S3_UNSAFE_ASSETS_DOMAIN_NAME` environment variable since that is not needed right now (we show a placeholder image instead).

### CODE REVIEW

File: apps/studio/next.config.mjs [hunk 1]
```diff
@@ -32,6 +32,7 @@ const ContentSecurityPolicy = `
     https://*.browser-intake-datadoghq.com
     https://vitals.vercel-insights.com/v1/vitals
     https://*.amazonaws.com
+    https://ohno.isomer.gov.sg
```
Comment (karrui) [inline apps/studio/next.config.mjs:35]: is this intended LOL

File: apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsImageControl.tsx [hunk 2]
```diff
@@ -51,6 +55,36 @@ export function JsonFormsImageControl({
...
+        : url
+      const file = await urlToFile(imageUrl, fileName || "", fileType)
+      setPendingAsset({ path, src: imageUrl, file })
+    }
+
+    if (!data) {
+      return
+    }
+
+    convertImage(data)
```
Comment (karrui) [inline apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsImageControl.tsx:83]: lint

File: apps/studio/next.config.mjs (hunk 1 above)
Comment (dcshzj) [reply apps/studio/next.config.mjs]: Oops removed.

File: apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsImageControl.tsx (hunk 2 above)
Comment (dcshzj) [reply apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsImageControl.tsx:84]: Disabled the eslint rule in d17406f2c80ade0533a78f923ca9d9c970556c61.

File: apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsImageControl.tsx [hunk 3]
```diff
@@ -13,11 +13,14 @@ import {
 import type { ModifiedAsset } from "~/types/assets"
 import { JSON_FORMS_RANKING } from "~/constants/formBuilder"
 import { useEditorDrawerContext } from "~/contexts/EditorDrawerContext"
+import { env } from "~/env.mjs"
 import {
   IMAGE_UPLOAD_ACCEPTED_MIME_TYPES,
   MAX_IMG_FILE_SIZE_BYTES,
 } from "./constants"
 
+const { NEXT_PUBLIC_S3_ASSETS_DOMAIN_NAME } = env
```
Comment (karrui) [inline apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsImageControl.tsx:22]: prefer using `useEnv` instead. better when needing to override in tests

**Review by karrui — APPROVED** [review]

**Pull request merged**